kebechet/managers/thoth_advise/thoth_advise.py

```diff
@@ -119,6 +120,18 @@ def _open_merge_request(
         else:
             body = DEFAULT_PR_BODY
 
+        with open(".thoth.yaml", "r") as f:
+            thoth_config = yaml.safe_load(f)
+        overlays_dir = thoth_config.get("overlays_dir")
+
+        full_name = (
+            f"{overlays_dir}/{self.runtime_environment}"
+            if overlays_dir
+            else self.runtime_environment
```

Comment:
how is `self.runtime_environment` passed to kebechet workflow? internal trigger that is using thoth database? 

isn't the thoth.yaml always more updated (cloned from repo when kebechet is scheduled right)? shouldn't we use the runtime_environment[0] by default if there are not overlay? which is the default behaviour usually when providing an advise?

Reply:
https://github.com/thoth-station/kebechet/pull/760/files/81075754a206a4de40dd1c5c73f014c1273a6c19#diff-2c3f14d5ab17b60e0aa85161f150995026a575029fefa506073b7087dd948611R254-R261

Here is where we set the default self.runtime_environment if it is not passed. If there is no overlays directory it defaults to the first entry in .thoth.yaml runtime environments